app/code/Magento/Sales/Test/Mftf/Test/StorefrontCustomerOrderDetailsPagerTest.xml

```diff
@@ -0,0 +1,286 @@
+            <argument name="product" value="$$createProduct16$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct17">
+            <argument name="product" value="$$createProduct17$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct18">
+            <argument name="product" value="$$createProduct18$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct19">
+            <argument name="product" value="$$createProduct19$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct20">
+            <argument name="product" value="$$createProduct20$$"/>
+        </actionGroup>
+        <!-- 1 configurable -->
+        <actionGroup ref="StorefrontAddConfigurableProductToTheCartActionGroup" stepKey="addConfigurableToCart">
+            <argument name="urlKey" value="$createConfigProduct.custom_attributes[url_key]$"/>
+            <argument name="productAttribute" value="$$createConfigProductAttribute.default_value$$"/>
+            <argument name="productOption" value="$$getConfigAttributeSecondOption.label$$"/>
+            <argument name="qty" value="1"/>
+        </actionGroup>
+        <!-- 1 bundle -->
+        <actionGroup ref="StorefrontOpenProductPageActionGroup" stepKey="openBundleProduct">
+            <argument name="productUrl" value="$$createBundleProduct.custom_attributes[url_key]$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddBundleProductToTheCartActionGroup" stepKey="addBundleProductToCart">
+            <argument name="productName" value="$simpleProduct2.name$"/>
+            <argument name="quantity" value="1"/>
+        </actionGroup>
+        <!-- Place the order -->
+        <actionGroup ref="StorefrontOpenCheckoutPageActionGroup" stepKey="onCheckout"/>
+        <see userInput="22" selector="{{CheckoutOrderSummarySection.itemsQtyInCart}}" stepKey="see21Products"/>
+        <click selector="{{CheckoutShippingSection.firstShippingMethod}}" stepKey="selectFirstShippingMethod"/>
+        <actionGroup ref="StorefrontCheckoutClickNextOnShippingStepActionGroup" stepKey="clickNextButton"/>
+        <click selector="{{CheckoutPaymentSection.placeOrder}}" stepKey="placeOrder"/>
+        <waitForPageLoad stepKey="waitForSuccess"/>
+        <grabTextFrom selector="{{CheckoutSuccessMainSection.orderNumber22}}" stepKey="grabOrderNumber"/>
+        <!-- Go to My Account > My Orders page -->
+        <actionGroup ref="StorefrontOpenMyAccountPageActionGroup" stepKey="onMyAccount"/>
+        <click selector="{{StorefrontCustomerSidebarSection.sidebarTab('My Orders')}}" stepKey="clickOnMyOrders"/>
+        <waitForPageLoad stepKey="waitForOrdersLoad"/>
```

Comment:
Create a new action group for this, it may be reusable.

Reply:
Replaced with the existing one.